Client/src/Utils/Theme/constants.js

```diff
@@ -15,7 +15,10 @@ const typographyLevels = {
 
 /* TODO Review color palette and semantic colors */
 const paletteColors = {
+	transparent: "#00000000",
 	white: "#FFFFFF",
+	white200:"#F6F6F6",
+	grey25: "#FCFCFC",
```

Comment:
This is almost the same as gray 90, use that instead

Reply:
sure